test/lib/tpunit++.cpp

```diff
@@ -396,6 +403,18 @@ void tpunit::TestFixture::tpunit_detail_do_tests(TestFixture* f) {
     }
 }
 
+void tpunit::TestFixture::testLog(const string& newLog) {
+    lock_guard<recursive_mutex> lock(*(_mutex));
+
+    // Format the buffer with an indent as we print it out.
+    testOutputBuffer = testOutputBuffer + "    " + newLog + "\n";
```

Comment:
This is slightly less code and slightly more efficient:
```
testOutputBuffer += "    " + newLog + "\n";
```

Reply:
done